contracts/rollup.sol

```diff
@@ -449,43 +447,14 @@ contract Rollup is RollupHelpers {
             bool
         )
     {
-        if (batchType == Types.Usage.Transfer) {
```

Comment:
@vaibhavchellani To merge the upstream we need to abort this part. But it's ok to do so since we are not using processBatch?

Reply:
We can do both. Lets quickly remove this and merge for now and later I can add this to upstream?

Reply:
Sounds good